packages/app/src/server/models/bookmark-folder.ts

```diff
@@ -96,21 +137,38 @@ bookmarkFolderSchema.statics.deleteFolderAndChildren = async function(boookmarkF
     }));
     const deletedChild = await this.deleteMany({ parent: bookmarkFolder });
     deletedCount += deletedChild.deletedCount + 1;
+    bookmarkFolder.delete();
   }
   return { deletedCount };
 };
 
 bookmarkFolderSchema.statics.updateBookmarkFolder = async function(bookmarkFolderId: string, name: string, parent: string):
```

Comment:
Small request. I think argument name `parent` an `id`, so args name is `parentId` easier to understand.